WordPress/Classes/Services/PostHelper.m

```diff
@@ -11,7 +11,7 @@ + (void)updatePost:(AbstractPost *)post withRemotePost:(RemotePost *)remotePost
     post.postID = remotePost.postID;
     // Used to populate author information for self-hosted sites.
     BlogAuthor *author = [post.blog getAuthorWithId:remotePost.authorID];
-
+    
```

Comment:
Leading whitespaces, _whelp_!

I'd love for SwiftLint to catch these for us... (I guess I can also make it happen)